### PR TITLE
[TensorV2] Add strided inputs and output

### PR DESCRIPTION
**Pull request by djeong20**
This PR adds the implementation of add operation that supports different strided inputs and output.

**Self-evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test:   [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
LGTM!

File: nntrainer/tensor/float_tensor.cpp [hunk 1]
```diff
@@ -476,6 +476,60 @@ TensorV2 &FloatTensor::divide(TensorV2 const &m, TensorV2 &output) const {
...
+                                   const float beta) const {
+  NNTR_THROW_IF(getData() == nullptr, std::invalid_argument)
+    << getName() << " is not allocated";
+  NNTR_THROW_IF(input.getData<float>() == nullptr, std::invalid_argument)
+    << input.getName() << " is not allocated";
+  NNTR_THROW_IF(output.getData<float>() == nullptr, std::invalid_argument)
+    << output.getName() << " is not allocated";
+
+  if (strides[3] != 1 || input.getStrides()[3] != 1 ||
+      output.getStrides()[3] != 1 || beta != 0.0) {
```
Comment (myungjoo) [inline nntrainer/tensor/float_tensor.cpp:489]: Are you really sure it's ok to use ```beta != 0.0``` with ```float beta```? 

Reference: https://stackoverflow.com/questions/4915462/how-should-i-do-floating-point-comparison

**Review by jihochu — APPROVED** [review]
LGTM

File: nntrainer/tensor/float_tensor.cpp (hunk 1 above)
Comment (djeong20) [reply nntrainer/tensor/float_tensor.cpp]: Good point. Fixed :)

**Pull request merged**